src/component/Footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const FooterData = [
```

Comment:
Nice use of an array to store the footer data - but don't capitalise this array because the convention is for functional components to be capitalised in React.
```suggestion
const footerData = [
```